Project.toml

```diff
@@ -12,16 +12,19 @@ Neighborhood = "645ca80c-8b79-4109-87ea-e1f58159d116"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+TimeseriesSurrogates = "c804724b-8c18-5caa-8579-6025a0767c70"
 
 [compat]
-DSP = "0.6.9"
+DSP = "^0.6.9, ^0.7"
 DelayEmbeddings = "2"
 Distances = "0.10"
 Entropies = "1"
 Neighborhood = "0.2"
 Reexport = "0.2, 0.3, 1"
 SpecialFunctions = "0.8, 0.9, 0.10, 1.2, 2"
 StaticArrays = "1"
+TimeseriesSurrogates = "^1.2.4"
```

Comment:
As a comment, the `^` is not necessary. The default behavior is to use `^`.

Reply:
Ok, I didn't catch that from the julia docs. Thanks!